apps/src/maze/maze.js

```diff
@@ -508,11 +519,22 @@ module.exports = class Maze {
 
     this.controller.animationsController.stopIdling();
 
+    if (this.controller.skin.adjustableSpeed) {
+      // The slider's value is between 1 and 0 with 1 being slowest for some reason.
+      // We make it between -1 and 1 with -1 being slowest by multiplying
+      // by 2, subtracting 1, then multiplying by -1.
+      this.stepSpeedMultiplier = Math.pow(
```

Comment:
the comment above doesn't seem to match the equation here, can you update the comment?